docs/admin/predeployment_check.md

```diff
@@ -289,6 +289,22 @@ commandline (including source builds), limits should be adjusted in
 [this article](https://access.redhat.com/solutions/61334) (which is a RHEL
 specific document but the instructions apply to most Linux distributions).
 
+### Memory mapped areas
+
+Low max number of per-process mapped areas (vm.max_map_count) can cause ULT
+stack allocation to fall-back from DAOS mmap()'ed way into Argobots preferred
+allocation method.
+
+For RPM installations, this is achieved through installed
```

Comment:
it isn't clear what "this" means --> expanding this value?

Reply:
You are right @Michael-Hennecke , in next push/commit, I will detail "this" to what I was thinking about when I wrote this ;-) Basically it is how vm.max_map_count default value is bumped to a much more higher value (1M) to better fit with the DAOS needs for the expected huge number of concurrent ULTs if we want all their stacks to be mmap()'ed.